src/components/AnnualReport/AGMDate.vue

```diff
@@ -54,15 +54,18 @@
             v-if="dateText"
             class="restriction-messages mt-4"
           >
-            <span v-if="!allowCoa && !allowCod">
+            <span
+              v-if="!allowCoa && !IsAuthorized(AuthorizedActions.ADDRESS_CHANGE_FILING) &&
+                !allowCod && !IsAuthorized(AuthorizedActions.DIRECTOR_CHANGE_FILING)"
+            >
               You cannot change your Registered Office Addresses or Directors in this Annual Report
               because your AGM predates another filing that may have conflicting changes.
             </span>
-            <span v-else-if="!allowCoa">
+            <span v-else-if="!allowCoa && !IsAuthorized(AuthorizedActions.ADDRESS_CHANGE_FILING)">
```

Comment:
This is a tricky one and you've probably thought a lot about it...

Do we actually need these checks here? This is a sub-component and the checks for the parent views should suffice, I think?

I'm also trying to wrap my head around what this check means. "If you're not allowed to change addresses and you're not allowed to file an Address Change filing." Seems redundant...?

Do you have a use case where this is needed?

Reply:
So, is this already handled in the Annual Report?

![image](https://github.com/user-attachments/assets/0bb1e88c-9020-4af9-81b0-cf72346670d1)

Reply:
(Actually, I think these checks aren't needed for an Annual Report. See my other comment.)

Reply:
Checks are removed. Resolving.